proto/babylon/epoching/v1/query.proto

```diff
@@ -224,6 +225,21 @@ message QueuedMessageResponse {
   // msg is the actual message that is sent by a user and is queued by the
   // epoching module as string.
   string msg = 5;
+  // enriched_msg contains a parsed, structured version of the original message
+  EnrichedMsg enriched_msg = 6;
```

Comment:
I think additional field could be something like `string queued_msg_type = 6` i.e we do not need the whole structure here but only full name of the type of the message being queued.